service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/utils/EncounterExtractor.java

```diff
@@ -19,7 +19,7 @@
 @NoArgsConstructor(access = AccessLevel.PRIVATE)
 @Component
 public final class EncounterExtractor {
-    private static final String CONSULTATION_LIST_CODE = "1149501000000101";
+    private static final String ENCOUNTER_LIST_CODE = "1149501000000101";
```

Comment:
Just a clarification requested by @mattd-kainos - otherwise doesn't impact this PR.